cypress/plugins/index.js

```diff
@@ -0,0 +1,17 @@
+// ***********************************************************
```

Comment:
Is a cypress change intentional?

Reply:
New cypress allows plugins, so checked in the empty plugins file in the event that we do use plugins in the future.  Prevents unchecked file locally.